lib/thumbs/pull_request_worker.rb

```diff
@@ -1,47 +1,66 @@
 module Thumbs
   class PullRequestWorker
     attr_reader :build_dir
-    attr_reader :build_status
```

Comment:
Several of these attr_reader and initialization in `initialize(options)` were removed to "lazily initialize". This technique is not direly needed, but is generally useful for approaching the class in testing and irb. The reasoning is that the state can be cached via instance variables (`@var ||= code_that_loads_the_state`). This technique makes invalidating the cached value easier, not requiring new'ing up the whole class instance again but also only loads state as needed.